src/result.ml

```diff
@@ -7,3 +7,8 @@ module Infix = struct
   let ( >>= ) = bind
   let ( >>| ) x f = map f x
 end
+
+module Syntax = struct
+  let ( let* ) = bind
+  let ( let+ ) x f = map f x
+end
```

Comment:
I like the Syntax (more than the Infix operators) -- maybe we should move the entire codebase to use only the syntax?

Reply:
Yea, I would like that as well. Maybe in another PR?

Reply:
yes